### ISSUE TITLE
Slider issues with ECS Pro 3.2 and Elementor  3.5.3

### ISSUE THREAD

**Issue by mikriz** [dudaster/ele-custom-skin#658]
1. when slider is activated and pause on hover is activated, button that trigger Elementor Popups fail (screenshot of error https://tinyurl.com/mu5rmbx4 ) turning off pause on hover removes error
2. slider navigation arrows missing on front end but work in elementor editor (screenshots https://tinyurl.com/2jjbyejt and https://tinyurl.com/yc3p539f)
3. Slider columns not changing or causing what should be 3 columns to just be 1 (no screenshots i noticed this issues posted already)

**Comment by designheroscotland** [comment]
I'm also experiencing these issues when using 2 sliders on page:
1. using custom loop causes elementor popup triggers to fail
2. custom loop slider not showing correct number of columns.

https://pasteboard.co/wDtXxsAh2AoK.png

**Comment by dudaster** [comment]
I think this is the same issue: #657

**Comment by mikriz** [comment]
yes it was thank you your last update fixed the JS error. BUT now i dont see the slider arrows (see screenshot https://tinyurl.com/yc2xwrkt and https://tinyurl.com/e7pffb4v ) i see them when im logged in but not when logged out. could this be a cache thing? ive cleared cache a ton.

**Comment by DigitalBerry** [comment]
Having the same issue here with the nav arrows missing on Live but visible in Elementor editor screen on this site - https://club44.co.uk/

Running latest versions of everything and no errors in Inspector.

> yes it was thank you your last update fixed the JS error. BUT now i dont see the slider arrows (see screenshot https://tinyurl.com/yc2xwrkt and https://tinyurl.com/e7pffb4v ) i see them when im logged in but not when logged out. could this be a cache thing? ive cleared cache a ton.

**Comment by mikriz** [comment]
I was able to fix this by turning off the Inline font icons experiment in Elementor settings

**Comment by kwsim539** [comment]
Just a shot in the dark here, but in your inspector, did you go to the Network tab, select Img, refresh the page and make sure the nav images are being called and they are returning a 200 status?

![image](https://user-images.githubusercontent.com/8410150/152552362-7aac7197-5dd1-43a0-a741-370db9221c6e.png)

**Comment by DigitalBerry** [comment]
> I was able to fix this by turning off the Inline font icons experiment in Elementor settings

This fixed it! Thanks @mikriz
